contents/teams/billing/objectives.mdx

```diff
@@ -0,0 +1,43 @@
+## Q2 2025 Objectives
+
+### Pato
+
+- Pricing changes
+    - API queries (new)
+    - Error tracking (new)
+    - Identified / Non identified (modification)
+    - LLM Observability (new)
+    - Experiments (new / modification)
+    - there may me more 
```

Comment:
We could update the list based on the latest we know from [here](https://posthog.slack.com/archives/C02E3BKC78F/p1742392042922429)?

Reply:
Things like CRM and Revenue analytics aren't likely to need pricing in Q2 since they will be beta.